### ISSUE TITLE
[Bug] Flickering blue artefacts along ringworld wall

### ISSUE THREAD

**Issue by ProPeach** [tsunamayo/Starship-EVO#3586]
### Short Description of the issue:
Flickering blue square pixels appear along the top of the ringworld. From asking around on Discord, it looks like this affects both AMD and Nvidea GPUs
https://i.imgur.com/VsLA7qP.mp4

### Steps to reproduce the issue:
These artefacts appear every game session, but are only visible if the player is outside of the ringworld atmosphere

### Environment
- Starship EVO version: 21w5b
- Graphics Card: RX 580

### Additional information:
Here's the same video as above zipped up for a higher quality - 
[Starship EVO Ringworld Artifacts.zip](https://github.com/tsunamayo/Starship-EVO/files/5937271/Starship.EVO.Ringworld.Artifacts.zip)

[Player.log](https://github.com/tsunamayo/Starship-EVO/files/5937278/Player.log)

**Comment by ProPeach** [comment]
My bad, looks like I've reported this already back in April with #1673. I knew I felt a little deja vu...

**Comment by Kaiser-Indrasil** [comment]
What's worrisome is that huge gap between the outside atmosphere shader and the inside one.
Recorded on an RTX 2080Ti:

https://user-images.githubusercontent.com/63756057/107125259-9b084a80-68a0-11eb-970e-83ebf7407c22.mp4

**Comment by tsunamayo** [comment]
Yep there is quite a few artifacts linked to the atmosphere... I keep you posted on that, it is on my to-do list since forever.

**Comment by tsunamayo** [comment]
To give you an idea, it is all using raytracing, there is a different drawing workflow when you are located inside, and to run faster I have to render it at half res and then upscale it. I think there are issues with all 3 =( Cheers
